src/modules/attempts/attempts.service.ts

```
@@ -638,4 +639,228 @@ export class AttemptsService {
+        }
+      }
+
+      // Only include correct answer information if showCorrectAnswer is enabled
+      if (test.showCorrectAnswer && question) {
+        // Add correct answer information based on question type
+        switch (question.type) {
+          case QuestionType.MCQ:
+          case QuestionType.TRUE_FALSE:
+          case QuestionType.MULTIPLE_ANSWER:
+            // Include correct options
+            const correctOptions = question.options?.filter(opt => opt.isCorrect) || [];
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)